package.json

```diff
@@ -51,5 +54,8 @@
   },
   "scripts": {
     "test": "grunt ci --verbose"
+  },
+  "browser": {
+    "pouchdb/lib/deps/request": "pouchdb/lib/deps/request-browser"
```

Comment:
I wonder, could this eventually break hoodie.js one day, when pouchDB releases a patch version, that moved `pouchdb/lib/deps/request-browser` into an own module? Anything we can do about it?